src/ziggurat/messaging/producer.clj

```diff
@@ -103,9 +103,20 @@
   "Get exponential backoff enabled for specific channel from config."
   (-> (ziggurat-config) :stream-router topic-entity :channels channel :retry :exponential-backoff-enabled))
 
-(defn- get-exponential-backoff [retry-count message-retry-count queue-timeout-ms]
+(defn- get-exponential-backoff [topic-entity channel message-payload]
```

Comment:
I have added a couple of comments on your last PR: https://github.com/gojek/ziggurat/commit/d3e437065549762c1f917d9c6457b7ebf8d72534

Can you push the fix for those with this PR as well?

Reply:
removed previous changes related with custom delay queue, this pr will be for exponential backoff only